NARFO_BE/NARFO_BE/Models/_Member.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace NARFO_BE.Models
+{
+    public class _Member
+    {
+        public int Id { get; set; }
+        public string MEMNO { get; set; }
+        public string Password { get; set; }
```

Comment:
Since this is not the password (I hope) but rather the hash, change the type to []byte (or whatever c# has), that way you can't save the clear text password (type string) as the hash (type bytes). 
This is optional.

Reply:
the string is a hashed string from the static method in the encription class....**thank you for your views (:**